app/services/idv_attempter.rb

```diff
@@ -0,0 +1,34 @@
+class IdvAttempter
+  attr_accessor :current_user, :user_session
```

Comment:
In this case, I don't think we need any `attr_accessors` because users of this class only need to access the `exceeded?` method, and internally, we can use instance variables in the initializer, with a private `attr_reader`.

```ruby
def initialize(current_user, user_session)
  @current_user = current_user
  @user_session = user_session
end

private

attr_reader :current_user, :user_session
```